src/components/pages/Categories/List.vue

```diff
@@ -52,12 +57,28 @@ export default {
     errorMessage() {
       return this.$store.state.category.errorMessage;
     },
+    deleteCategoryName() {
+      return this.$store.state.category.deleteCategory.name;
+    },
   },
   created() {
     this.$store.dispatch('category/getAllCategories');
     this.$store.dispatch('category/clearMessage');
   },
   methods: {
+    openModal(categoryId, categoryName) {
+      const payload = {
+        categoryId,
+        categoryName,
+      };
+      this.$store.dispatch('category/confirmDeleteCategory', payload);
+      this.toggleModal();
```

Comment:
[nits]

一時変数は逆に可読性を落としてしまったり、変数を新たに定義するのでメモリの消費にもつながります！
引数に直接指定しまって良いかなと思います！

```suggestion
      this.$store.dispatch('category/confirmDeleteCategory', {
        categoryId,
        categoryName,
      });
      this.toggleModal();
```

Reply:
ありがとうございます！
こちらの方が記述が完結ですし、わかりやすかったので、変更させていただきました！